include/grpc++/credentials.h

```diff
@@ -65,78 +73,78 @@ class Credentials : public GrpcLibrary {
       const grpc::string& target, const ChannelArguments& args) = 0;
 };
 
-// Options used to build SslCredentials
-// pem_roots_cert is the buffer containing the PEM encoding of the server root
-// certificates. If this parameter is empty, the default roots will be used.
-// pem_private_key is the buffer containing the PEM encoding of the client's
-// private key. This parameter can be empty if the client does not have a
-// private key.
-// pem_cert_chain is the buffer containing the PEM encoding of the client's
-// certificate chain. This parameter can be empty if the client does not have
-// a certificate chain.
+/// Options used to build SslCredentials.
+///
+/// pem_roots_cert is the buffer containing the PEM encoding of the server root
+/// certificates. If this parameter is empty, the default roots will be used.
+/// pem_private_key is the buffer containing the PEM encoding of the client's
+/// private key. This parameter can be empty if the client does not have a
+/// private key.
+/// pem_cert_chain is the buffer containing the PEM encoding of the client's
+/// certificate chain. This parameter can be empty if the client does not have
+/// a certificate chain.
 struct SslCredentialsOptions {
   grpc::string pem_root_certs;
   grpc::string pem_private_key;
   grpc::string pem_cert_chain;
 };
 
-// Factories for building different types of Credentials
-// The functions may return empty shared_ptr when credentials cannot be created.
-// If a Credentials pointer is returned, it can still be invalid when used to
-// create a channel. A lame channel will be created then and all rpcs will
-// fail on it.
+// Factories for building different types of Credentials The functions may
+// return empty shared_ptr when credentials cannot be created. If a
+// Credentials pointer is returned, it can still be invalid when used to create
+// a channel. A lame channel will be created then and all rpcs will fail on it.
 
-// Builds credentials with reasonable defaults.
+/// Builds credentials with reasonable defaults.
```

Comment:
Please add:
/// WARNING: Only use these credentials when connecting to a google endpoint.
/// Using these credentials to connect to any other service may result in this service
/// being able to impersonate your client for requests to google services.